src/components/NcAvatar/NcAvatar.vue

```diff
@@ -370,11 +370,8 @@ export default {
 	},
 	computed: {
 		avatarAriaLabel() {
-			if (!this.hasMenu) {
-				return
-			}
```

Comment:
We'd need to keep this as non-interactive elements should not have an `aria-label`, added in https://github.com/nextcloud-libraries/nextcloud-vue/pull/3618

Reply:
Reverted and added a comment about the purpose of that check :)